app/views/group_plan_summary.scala.html

```diff
@@ -14,74 +14,106 @@
+@import _root_.utils.ERSUtil
 @import config.ApplicationConfig
-@import utils.CountryCodes
+@import _root_.utils.CountryCodes
 @import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+@import views.html.templates.reference
+@import views.html.govuk_wrapper
 
-@this(form: FormWithCSRF)
+@this(
+		govuk_wrapper: govuk_wrapper,
+		govukBackLink: GovukBackLink,
+		govukSummaryList: GovukSummaryList,
+		govukButton: GovukButton,
+		hmrcPageHeading: HmrcPageHeading,
+		formWithCSRF: FormWithCSRF
+)
 
 @(requestObject: RequestObject, groupSchemeType: String, companies: CompanyDetailsList)(implicit request: Request[AnyRef], flash: Flash, messages: Messages, countryCodes: CountryCodes, ersUtil: ERSUtil, appConfig: ApplicationConfig)
-@* Is a group scheme template *@
+
 @schemeId = @{requestObject.getSchemeId}
-@import views.html.templates._
 
-@ers_main(title = ersUtil.getPageElement(schemeId, "ers", "group_summary.page_title") + " " + messages("ers.title.postfix")) {
+@govuk_wrapper(title = ersUtil.getPageElement(schemeId, "ers", "group_summary.page_title") + " " + messages("ers.title.postfix")) {
 
 	@reference(requestObject.getPageTitle)
 
-	<p class="font-xsmall"><a href="@ersUtil.getPageBackLink(schemeId,ersUtil.PAGE_GROUP_SUMMARY,groupSchemeType)" class="link-back">@Html(messages("ers.back"))</a></p>
+	@govukBackLink(BackLink(
+		href = ersUtil.getPageBackLink(schemeId,ersUtil.PAGE_GROUP_SUMMARY,groupSchemeType),
+	))
+
+	@hmrcPageHeading(PageHeading(
+		text = ersUtil.getPageElement(schemeId, ersUtil.PAGE_GROUP_SUMMARY, "title")
+	))
+
+	@formWithCSRF(action = routes.GroupSchemeController.groupPlanSummaryContinue(schemeId)) {
 
-	@header(ersUtil.getPageElement(schemeId, ersUtil.PAGE_GROUP_SUMMARY, "title"))
+		<a class="govuk-link govuk-inset-text govuk-!-display-inline-block govuk-!-margin-top-0 govuk-!-width-full" href="@routes.GroupSchemeController.manualCompanyDetailsPage()">@ersUtil.getPageElement(schemeId, ersUtil.PAGE_GROUP_SUMMARY, "add_company")</a>
 
-	<div class="clear">
-		@form(action = routes.GroupSchemeController.groupPlanSummaryContinue(schemeId)) {
-			@if(groupSchemeType == ersUtil.OPTION_MANUAL) {
-			<div class="panel panel-border-wide">
-				<a href="@routes.GroupSchemeController.manualCompanyDetailsPage()">@Html(ersUtil.getPageElement(schemeId, ersUtil.PAGE_GROUP_SUMMARY, "add_company"))</a>
-			</div>
-			}
-			@for((company, index) <- companies.companies.zipWithIndex) {
-			<table>
-			  <thead>
-				<tr>
-					<th scope="col" id="group-plan-summary-name">@company.companyName</th>
-					<th id="group-plan-summary-address" class="group-plan-summary-action-text-align"><a href="@routes.GroupSchemeController.editCompany(index)">@Html(messages("ers.change_this"))</a>&nbsp;&nbsp;&nbsp;<a href="@routes.GroupSchemeController.deleteCompany(index)">@Html(messages("ers.delete"))</a></th>
-				</tr>
-			  </thead>
-			  <tbody>
+		@for((company, index) <- companies.companies.zipWithIndex) {
+			@govukSummaryList(SummaryList(
```

Comment:
not sure if this is something we can avert, but axe is finding an issue here (badly ordered contents inside the <dl>)

Reply:
This is following the SummaryList pattern so not sure there is anything I can change? Key -> Value -> Action seems logical.